examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -103,3 +95,7 @@ Where
 This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console.
 
 Please follow the sections ["Setting up the training environment on trn1.32xlarge"](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/tutorials/training/bert.html#setting-up-the-training-environment-on-trn1-32xlarge) and ["Downloading tokenized and sharded dataset files"](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/tutorials/training/bert.html#downloading-tokenized-and-sharded-dataset-files) to setup the BERT scripts and download the dataset files.
```

Comment:
Remove this line. They are included in https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/blob/master/examples/jobs/dp-bert-launch-job.md .

Add pointer to https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/blob/master/examples/jobs/dp-bert-launch-job.md after step #3 .

Reply:
Remove 'Please follow the sections ["Setting up the training environment on trn1.32xlarge"](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/tutorials/training/bert.html#setting-up-the-training-environment-on-trn1-32xlarge) and ["Downloading tokenized and sharded dataset files"](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/tutorials/training/bert.html#downloading-tokenized-and-sharded-dataset-files) to setup the BERT scripts and download the dataset files.'

Reply:
added pointer after 3: 
4. After post-installation actions are complete, see [this](../jobs/dp-bert-launch-job.md) about how to launch a training job.